services/spark-operator/metadata.yaml

```diff
@@ -0,0 +1,11 @@
+displayName: Spark
+description: Apache Spark is a fast and general-purpose cluster computing system for big data
+category:
+  - compute
+  - bigdata
+overview: >
+  # Overview
+  Apache Spark is a fast and general-purpose cluster computing system for big data. It provides high-level APIs in Scala, Java, Python, and R, and an optimized engine that supports general computation graphs for data analysis. It also supports a rich set of higher-level tools including: Spark SQL for SQL and DataFrames, MLlib for machine learning, GraphX for graph processing, and Spark Streaming for stream processing.
+
+  - [Spark Documentation](https://spark.apache.org/docs/latest/)
```

Comment:
Does this catalog item always install the 'latest' Spark or does it install a particular version?   If so, where is the Version that is being installed displayed?   

If we're installing a particular version, then I think the link here should point at the docs for whatever version is being installed.   For example, for Spark 3.0.3, the link is:

https://spark.apache.org/docs/3.0.3/index.html

Reply:
Catalog can have many different versions. We don't have distinctive per-version descriptions yet, so this same page will show for all versions for now. We have a [story](https://jira.d2iq.com/browse/D2IQ-80874) to look into supporting different details per version but it's not implemented yet. Once that's implemented, we can include version-specific info